doc/cli.markdown

```diff
@@ -15,27 +15,26 @@ Check the [balena CLI installation instructions on GitHub](https://github.com/ba
 
 ### Choosing a shell (command prompt/terminal)
 
-On Linux and MacOS, `bash` is the standard and recommended shell for use with the balena CLI.
-
-> **bash command auto completion**  
-`bash` command auto completion can be enabled by copying the
-[balena-completion.bash](https://github.com/balena-io/balena-cli/blob/master/balena-completion.bash)
-file to the default bash completions directory (usually `/etc/bash_completion.d/`), or by appending
-it to `~/.bash_completion`.
-
-On Windows, we support the standard Windows Command Prompt (`cmd.exe`) and the Windows
-[PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/getting-started-with-windows-powershell?view=powershell-6).
-We are aware of users also having a good experience with alternative shells, including:
+On **Windows,** the standard Command Prompt (`cmd.exe`) and
+[PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/getting-started-with-windows-powershell?view=powershell-6)
+are supported. We are aware of users also having a good experience with alternative shells,
+including:
 
 * Microsoft's [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about)
   (a.k.a. Microsoft's "bash for Windows 10").
 * [Git for Windows](https://git-for-windows.github.io/).
 * [MinGW](http://www.mingw.org): install the `msys-rsync` and `msys-openssh` packages too.
 
+On **macOS** and **Linux,** the standard terminal window is supported. _Optionally,_ `bash` command
+auto completion may be enabled by copying the
+[balena-completion.bash](https://github.com/balena-io/balena-cli/blob/master/balena-completion.bash)
+file to your system's `bash_completion` directory: check [Docker's command completion
+guide](https://docs.docker.com/compose/completion/) for system setup instructions.
```

Comment:
* The previous wording _("`bash` is the standard and recommended shell for use with the balena CLI")_ was not obvious enough that macOS and Linux users don't need to do anything (consider that some users may not know what `bash` is). The new wording _("the standard terminal window is supported")_ should be clearer on that point.
* On bash autocompletion, the previous instructions _("copy the file to the default bash completions directory, usually `/etc/bash_completion.d/`")_ failed to address the fact that macOS does not have that directory by default, and requires some `homebrew` installation to achieve it. The new instructions simply point to Docker's command completion guide, who have lots of details on it (check the link!), while stressing it is an optional step.
* All other changes in this PR were requested by @hedss in a late review to #1175.